orchestration/values.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "gcs": {
+      "type": "object",
+      "properties": {
+        "bucketName": { "type": "string" }
+      },
+      "required": ["bucketName"]
+    },
+    "serviceAccount": {
+      "type": "object",
+      "properties": {
+        "k8sName": { "type": "string" },
+        "googleName": { "type": "string" }
+      },
+      "required": ["k8sName", "googleName"]
+    },
+    "cron": {
+      "type": "object",
+      "properties": {
+        "enable": { "type": "boolean" },
+        "schedule": { "type": "string" },
+        "timezone": { "type": "string" }
+      },
+      "required": ["enable", "schedule"]
+    },
+    "dataflow": {
+      "type": "object",
+      "properties": {
+        "project": { "type": "string" },
+        "region": { "type": "string" },
+        "tmpBucketName": { "type": "string" },
+        "subnetName": { "type": "string" },
+        "workerAccount": { "type": "string" },
+        "workerMachineType": { "type": "string" },
+        "autoscaling": {
+          "type": "object",
+          "properties": {
+            "minWorkers": {
+              "type": "integer",
+              "minimum": 1
+            },
+            "maxWorkers": {
+              "type": "integer",
+              "minimum": 1
+            }
+          },
+          "required": ["minWorkers", "maxWorkers"]
+        }
+      },
+      "required": ["project", "region", "tmpBucketName",
+        "subnetName", "workerAccount", "workerMachineType",
+        "autoscaling"]
+    }
+  },
+  "redcap": {
+    "type": "object",
+    "properties": {
+      "apiKey": { "type": "string" }
```

Comment:
This structure isn't secure. Instead of type `string`, this should be an object with keys:
* `secretName`
* `secretKey`